### DISCUSSION TITLE
Unable to find program 'ar' from certs_from_mozilla.py Exception

### DISCUSSION THREAD

**Discussion by garretj99** [esp8266/Arduino#8375] [Q&A]
Hello,

When attempting to execute certs_from_mozilla.py in my IDE, an Exception is raised which says:

"You need the program 'ar' from xtensa-lx106-elf found here: (esp8266-arduino-core)/hardware/esp8266com/esp8266/tools/xtensa-lx106-elf/xtensa-lx106-elf/bin/ar"). 

I am fairly new to Git so forgive me if I am out of the loop on something, but how to I access "here" and find this program? I am not sure what to do with that path and it hasn't returned relevant results in the Github search bar. Any help is appreciated. Thanks.

**Comment by icwilson20** [comment]
I had this issue and I fixed it by following the steps here: https://community.hivemq.com/t/getting-start-with-arduino-esp8266-certificate-generation-error/844/16

Best of luck.
